### DISCUSSION TITLE
integrated SeleniumBase with Azure DevOps?

### DISCUSSION THREAD

**Discussion by maimul** [seleniumbase/SeleniumBase#1881] [Q&A] [answered]
Hi! Hope you are well. I would like to automate some test cases and integrate it with Azure DevOps. I am new to SeleniumBase and I would like to know if it is possible to integrate SeleniumBase with Azure DevOps. Below is an example of the workflow that I am expecting.

Basically every time a codebase is committed to test branch, the Azure pipeline would trigger the test automation and generate a report.

If anyone have any suggestions or have experience with Azure DevOps with Selenium, could you please share your tips and tricks.

Thank you! Appreciate your help.

**Comment by mdmintz** [comment, marked as answer]
Hello, and welcome! There’s documentation for running SeleniumBase tests using Azure DevOps here: https://github.com/seleniumbase/SeleniumBase/tree/master/integrations/azure/azure_pipelines
